magento/pwa-studio
# [PWA-1030] Merge Cart does not update mini cart if auth user has same simple product already added

## Description

URL - [https://develop.pwa-venia.com/]

Steps - 
1. Login as Auth user
2. add a simple product or config product to cart
3. Sign out from account.
4. Now add same simple or config product with same configs to gues cart
5. Now login to as Auth user again

Expected - Mini cart count should update and if user open mini cart it should show updated counts.

Actual - Mini cart count does not update. 

Note - 
1. Cart page works fine
2. If user does browser refresh then mini cart updates.
3. Works fine if guest cart has any item other than auth cart has.

## Related Issue





* [[PWA-1030](https://jira.corp.magento.com/browse/PWA-1030)] Merge Cart does not update mini cart if auth user has same simple product already added

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Run through reproduction steps, verify that counts are now accurate (compare to counts after refresh)

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
